.husky/post-merge

```diff
@@ -11,7 +11,7 @@ changedNode="$(echo "$changedFiles" | { grep '.nvmrc' || :; })"
 if [[ ! -z $changedLock ]]
 then
 	echo "${RED}This application has new dependencies. Running 'yarn'... ${ENDCOLOR}"
-	yarn
+	yarn --frozen-lockfile
```

Comment:
i'm not 100% sure this is needed? DCR always installs workspace deps when running tasks, so this is just for apps i think? and AR deps are only installed by this because of `post-install` hook in the workspace, of which AR is not a member.

is there any value in installing AR deps when installing DCR deps?

Reply:
The value was when we had a shared storybook, you needed _all_ the dependencies to be up-to-date in order to run storybook. So you really wanted to be able to run `yarn` in the root directory and not have to remember to go check your AR dependencies were up-to-date as well before running storybook.

I'm guessing this is no longer needed, since we now have separate storybooks, and you'd be good to remove the post-install?

Reply:
thanks @OllysCoding – will do in 2nd PR